docs/UserGuide.md

```diff
@@ -256,13 +256,24 @@ Example:
 
 ### Room Management
 
+<div markdown="block" class="alert alert-secondary">
+:thinking: Why do room commands start with `o`?<br>
+:point_right: Room commands are prefixed with `o` as `r` is taken up by Resident commands. `o` is the second character in R**o**om. 
+</div>
+
 #### Add a room : `oadd`
 
 Adds a room to the housing management system.
 
 Format: `oadd r/ROOM_NUMBER t/ROOM_TYPE [g/TAG]`
 * Room is initialised with default occupancy status of "No".
-* Room occupancy status can only be changed through the `alloc` or `dealloc` command when a resident is allocated or deallocated. The occupancy status cannot be defaulted to "Yes" during room addition.
+* The occupancy status cannot be defaulted to "Yes" during room addition.
+* Room occupancy status can only be changed through the `alloc` or `dealloc` command when a resident is allocated or deallocated. See [allocate a resident](#allocate-resident-to-room-alloc) or [deallocate a resident](#deallocate-resident-from-room--dealloc) for more info. 
```

Comment:
The `dealloc` link seems to not work for me.

Reply:
Fixed